docs/src/inputs/input/examples/exampleInputOnClick.jsx

```diff
@@ -0,0 +1,14 @@
+import { Input } from 'react-cm-ui';
+import React from 'react';
+
+function ExampleInputOnClick() {
+    const onInputClick = () => {
+        window.alert('You did it! You clicked the input.');
```

Comment:
I think we have to suppress an ESLINT `no-alert` rule?

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef

Reply:
Looks good.  Resolving.